proxima-chapter-1.md

```diff
@@ -1,17 +1,16 @@
 ---
-title: Chapter One: setting up the event-driven OCI components
-categories: [CATEGORIES]
-tags: [PROXIMA]
+title: "Chapter One: setting up the event-driven OCI components"
+categories: [cloudapps]
+tags: [devops,fullstack,oci,hardware]
 thumbnail: assets/promima-safe-chapter1-4.jpg
+parent: Wedo
```

Comment:
As mentioned, tags/categories are documented in the README. They're not complete yet, though, so you're welcome to suggest new ones as needed. 

The `parent` key is vital, without it the page won't show up in any navigation. In the `index.md` file for the parent project there's a key called `project`. The `parent` value must match the value of the project's `project` key.